src/lib/swapper/swappers/ThorchainSwapper/buildThorTrade/buildThorTrade.ts

```diff
@@ -75,6 +75,8 @@ export const buildTrade = async (
   if (chainNamespace === CHAIN_NAMESPACE.Evm) {
     const evmQuote = quote as ThorEvmTradeQuote
 
+    // TODO(gomes): it is not - plumb from through
```

Comment:
Actual TODO, to be akschually tackled in a follow-up, reduces the risk area like so